services/movie.py

```diff
@@ -0,0 +1,36 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> Movie:
+    queryset = Movie.objects.all()
+
+    if not genres_ids and not actors_ids:
+        return queryset
+
+    if genres_ids and actors_ids:
+        return queryset.filter(genres__in=genres_ids, actors__in=actors_ids)
+
+    if genres_ids:
+        return queryset.filter(genres__in=genres_ids)
+
+    if actors_ids:
+        return queryset.filter(actors__in=actors_ids)
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(movie_title: str,
+                 movie_description: str,
+                 genres_ids: list[int] = None,
+                 actors_ids: list[int] = None) -> None:
```

Comment:
The return type for `create_movie` function should be `Movie` instead of `None`, because the function is creating and returning a new movie instance. Please, correct the return type.